src/os/CMakeLists.txt

```diff
@@ -25,6 +26,9 @@ if(WITH_BLUESTORE)
     bluestore/BtreeAllocator.cc
     bluestore/HybridAllocator.cc
   )
+  list (APPEND libos_exp_srcs
```

Comment:
How about commenting here that we are creating `bluestore-exp`, an offshoot of stable bluestore intented for baking in potentially hazardous features?